### ISSUE TITLE
Merge with vscode-elixir

### ISSUE THREAD

**Issue by fr1zle** [JakeBecker/vscode-elixir-ls#8]
Hey Jake,

I just found your extension and really like the fact that you are going with the language server protocol. 
If you ever feel like the language server is stable enough and you are willing to, I would love to merge your work to vscode-elixir and making you a contributor.
For discoverability this will benefit most users i think!

**Comment by JakeBecker** [comment]
Hi! Thanks for getting in touch!

I would probably be down for something like that once ElixirLS is more stable. I found your plugin very helpful while writing ElixirLS, and I copied the syntax highlighting directly from it. Great work on it!

The biggest impediment to calling ElixirLS "stable" right now is the build system, and I'm working with the Elixir folks to make compilation work more nicely with external tools. Once that's improved, we should talk more about merging our plugins.
